.changeset/other-test.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/generator': minor
+---
+
+[Compatibility Notes] Deprecate the `generatePackageJson` option. Use `packageJson` instead.
```

Comment:
I assume these test changeset files will be removed before the PR is merged. Otherwise I think they will pollute our release notes...

Reply:
Yes, I forgot to remove them before.